server/nginx/conf/nginx.conf

```diff
@@ -0,0 +1,92 @@
+worker_processes 1;
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+    sendfile        on;
+    keepalive_timeout  65;
+
+    server {
+        server_name  auth.averycb.net;
+
+        location ~ /\.well-known/acme-challenge/ {
+            allow all;
+            root /var/www/letsencrypt;
+	    try_files $uri =404;
+            break;
+        }
+
+    	proxy_set_header X-Forwarded-For $proxy_protocol_addr; # To forward the original client's IP address 
+	proxy_set_header X-Forwarded-Proto $scheme; # to forward the  original protocol (HTTP or HTTPS)
+	proxy_set_header Host $host; # to forward the original host requested by the client
+
+
+        location ~ /* {
+            proxy_pass http://keycloak:8080;
+        }
+    
+        listen 443 ssl; # managed by Certbot
+        ssl_certificate /etc/letsencrypt/live/api.averycb.net/fullchain.pem; # managed by Certbot
```

Comment:
Ditto for this stuff, maybe we can split into a production and dev nginx conf next sprint?